config/webpack/terra-dev-site.webpack.config.js

```diff
@@ -42,13 +43,36 @@ const aliasMonoRepoPackages = (monoRepo, hotReloading, webpackAliasOptions = {},
   }, {});
 };
 
+/**
+* Create index.html files for terra-dev-site and the additional apps.
+*/
+const indexPlugin = ({
+  title, filename, lang, rootElementId = 'root', siteConfig, indexEntryPoints, entry,
+}) => {
+  const otherIndexChunks = indexEntryPoints.filter(indexEntry => indexEntry !== entry);
+  return new HtmlWebpackPlugin({
+    title,
+    filename,
+    template: path.join(__dirname, '..', '..', 'lib', 'index.html'),
+    lang,
+    rootElementId,
+    dir: siteConfig.appConfig.defaultDirection,
+    favicon: siteConfig.appConfig.favicon,
+    headHtml: [getNewRelicJS()].concat(siteConfig.appConfig.headHtml),
+    headChunks: ['rewriteHistory'],
+    excludeChunks: ['redirect', ...otherIndexChunks],
+    inject: false, // This turns off auto injection. We handle this ourselves in the template.
+  });
+};
+
 /**
 * Generates the file representing app name configuration.
 */
 const devSiteConfig = (env = {}, argv = {}) => {
   const production = argv.p;
   const processPath = process.cwd();
   const verbose = env.verboseGenerateAppConfig;
+  const publicPath = argv['output-public-path'] || process.env.TERRA_DEV_SITE_PUBLIC_PATH || '/';
```

Comment:
By default wepback defines the public path as `''`. Are we setting this to `/` for code splitting?

Reply:
we set it to slash for browser routing, I want assets pulled from root not relative current location.